docs/src/guides/common.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Hub.js Utilities
+navTitle: Utilities
+description: Learn leverage the lowest-level utilities that power @esri/hub.js.
+order: 60
+group: 2-concepts
+instanceColor: '{% raw %}{{instance.color}}{% endraw %}'
+outputItemId: '{% raw %}{{outputItem.id}}{% endraw %}'
+teamsCollaborationId: '{% raw %}{{teams.collaborationId}}{% endraw %}'
```

Comment:
This is sooo messy, but the only way I could figure to get `{{ thing }}` into the output of a markdown file that's also processed by nunjucks. Glad we don't have to do a lot of examples of this